src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Errors/ErrorFmt.cs

```diff
@@ -76,35 +76,35 @@ public ErrArg()
         }
         public ErrArg(int n)
         {
-            this.eak = ErrArgKind.Int;
-            this.eaf = ErrArgFlags.None;
```

Comment:
Really, it should be `Eak` because it's public, and should probably be an auto-property (I don't think there's any reason it has to be a field, but haven't yet checked). That's a wider-impacting change so for now I'm just reverting all the `this.` removals when the name involved is lower-case and not a method invocation (I suppose those could be invoked delegates, but there aren't many such delegates around to cause such confusion).